pxr/usd/usdGeom/mesh.h

```diff
@@ -228,7 +228,7 @@ class UsdGeomMesh : public UsdGeomPointBased
     // --------------------------------------------------------------------- //
     // FACEVERTEXINDICES 
     // --------------------------------------------------------------------- //
-    /// Flat list of the index (into the 'points' attribute) of each
+    /// Flat list of the index (into the _points_ attribute) of each
     /// point of each face in the mesh.  If this attribute has more than
```

Comment:
Given what we've said above, and the *name* of this attribute, would it not be more appropriate to use `vertex` here instead of `point` , since the vertex is already in reference to the face?

Reply:
Agreed, I was trying to strictly avoid "vertex" -- except as the token for the interpolation type -- but it makes sense here.  Will include in another commit.